frontend/src/scenes/funnels/funnelLogic.ts

```diff
@@ -480,7 +480,7 @@ export const funnelLogic = kea<funnelLogicType>({
             personsModalLogic.actions.loadPeople({
                 action: { id: step.action_id, name: step.name, properties: [], type: step.type },
                 breakdown_value: breakdown_value || '',
-                label: `Persons who completed Step #${stepNumber} - "${step.name}"`,
+                label: `Persons who dropped off at Step #${stepNumber} - "${step.name}"`,
```

Comment:
Hey @Patil2099 - you'd also need to check what `stepNumber` is. If it's positive, completed is correct. If it's negative, dropping off is correct.

Reply:
Sure

Reply:
I will make the changes

Reply:
@patil2099 sorry my comment was incorrect, Neil's comment is correct!

Reply:
"Persons who **completed** Step X" and "Persons who **dropped off at** Step Y" , yes :)

Reply:
(and then, also `Maths.abs` on the number :) )

Reply:
Hey, just unresolved this, have a look at the two comments here^^

Reply:
`Persons who ${stepNumber >= 0? 'completed': 'dropped off'} at Step #${stepNumber} - "${step.name}"`,

Reply:
Is this correct ?